### PR TITLE
Upgrade to Scala 2.13 (#3).

### PR DESCRIPTION
**Pull request by tzeman77**
- finagle -> 21.2.0
- sbt -> 1.4.7
- no guava dependency
- refactored to use json4s instead of rapture-json
- fixed handling of NULL values for insert/update
- minor fixes of formatting/code style/type annotation

### CODE REVIEW

File: src/main/scala/reson/server/Server.scala [hunk 1]
```diff
@@ -1,53 +1,64 @@
...
+ */
 object Server extends App {
-  def handle(req: Request, table: String, f: String => Future[String]): Future[Response] = {
+  def handle(req: Request, table: String,
+    f: String => Future[JValue]): Future[Response] = {
     for {
       query <- RequestParser.parse(req, table).toFuture
-      qString <- query.materialize.toFuture
-      dbResp <- f(qString)
+        qString <- query.materialize.toFuture
```
Comment (sscarduzio) [inline src/main/scala/reson/server/Server.scala]: what's the deal with these spaces?

File: src/main/scala/reson/server/Server.scala [hunk 2]
```diff
@@ -1,53 +1,64 @@
...
+        dbResp <- f(qString)
     } yield mkResp(dbResp)
   }
 
   lazy val route = Service.mk[Request, Response] { req =>
     (req.method, Path(req.path)) match {
       case (Get, Root) => MySQL.tableList.map(mkResp)
-      case (Get | Options, Root / (table: String)) => handle(req, table, MySQL.read)
+      case (Get | Options, Root / (table: String)) => handle(req, table,
+        MySQL.read)
```
Comment (sscarduzio) [inline src/main/scala/reson/server/Server.scala]: are you using a very narrow line length?

**Review by sscarduzio — COMMENTED** [review]
Just cosmetic comments

File: src/main/scala/reson/server/Server.scala (hunk 1 above)
Comment (tzeman77) [reply src/main/scala/reson/server/Server.scala]: I'm sorry that was oversight on my part; IdeaJ reformatted the code and obviously, all rows inside for {} block should be aligned. Will fix and refresh pull request.

File: src/main/scala/reson/server/Server.scala (hunk 2 above)
Comment (tzeman77) [reply src/main/scala/reson/server/Server.scala]: I'm using std. 80 chars line length... I can revert this change if you like.

Comment (sscarduzio) [reply src/main/scala/reson/server/Server.scala]: yes please set it at least 2x (160 chars) for Scala! thanks

Comment (tzeman77) [reply src/main/scala/reson/server/Server.scala]: Ok + reverted line wraps to be on par with original code.

**Comment by tzeman77** [comment]
It may be worth squashing both commits into one.

File: src/main/scala/reson/server/Server.scala (hunk 2 above)
Comment (sscarduzio) [reply src/main/scala/reson/server/Server.scala]: Thank you @tzeman77! Good job :)

**Pull request merged**